pubspec.yaml

```diff
@@ -9,40 +9,41 @@ executables:
   sass: sass
 
 environment:
-  sdk: '>=2.6.0 <3.0.0'
+  sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
-  args: ">=1.4.0 <3.0.0"
-  async: ">=1.10.0 <3.0.0"
-  charcode: "^1.1.0"
-  cli_repl: ">=0.1.3 <0.3.0"
-  collection: "^1.8.0"
-  meta: "^1.1.7"
-  node_interop: "^1.1.0"
-  js: "^0.6.0"
-  package_resolver: "^1.0.0"
-  path: "^1.6.0"
-  source_maps: "^0.10.5"
-  source_span: "^1.6.0"
-  stack_trace: ">=0.9.0 <2.0.0"
-  stream_transform: ">=0.0.20 <3.0.0"
-  string_scanner: ">=0.1.5 <2.0.0"
-  term_glyph: "^1.0.0"
-  tuple: "^1.0.0"
-  watcher: ">=0.9.6 <2.0.0"
+  args: ^2.0.0
+  async: ^2.5.0
+  charcode: ^1.2.0
+  cli_repl: ^0.2.1-nullsafety
+  collection: ^1.15.0
+  meta: ^1.3.0
+  node_interop: ^2.0.0
+  js: ^0.6.3
+  path: ^1.8.0
+  source_maps: ^0.10.10
+  source_span: ^1.8.1
+  stack_trace: ^1.10.0
+  stream_transform: ^2.0.0
+  string_scanner: ^1.1.0
+  term_glyph: ^1.2.0
+  tuple: ^2.0.0-nullsafety.0
+  watcher: ^1.0.0
+
+publish_to: none
```

Comment:
This needs to be removed before publishing. I think prerelease versions for dependencies also need to be removed to publish a stable version.